prober/prober.dockerfile

```diff
@@ -0,0 +1,15 @@
+# Base image that supports bash
+FROM cgr.dev/chainguard/bash:latest
+
+COPY jvsctl /jvsctl
+COPY ./prober/prober.sh prober.sh
+
+# Normally we would set this to run as "nobody".
+# But goreleaser builds the binary locally and sometimes it will mess up the permission
+# and cause "exec user process caused: permission denied".
+#
+# USER nobody
+
+# Run the web service on container startup.
+
+ENTRYPOINT ["bash", "prober.sh"]
```

Comment:
What will happen without `bash` here? Can't we use `prober.sh` as the entry point?

Reply:
It will have a permission denied error whens starting the container: `terminated: Application failed to start: invalid status ::14: could not start container: permission denied`.

I think we don't want to run `chmod` command on the script to solve this problem. So I chose to use `ENTRYPOINT ["bash", "prober.sh"]`

Reply:
> I think we don't want to run chmod command on the script to solve this problem

Even if it's not this problem, we should `chmod +x` to make the script executable. If it solves the problem, even better.

Reply:
Done.